core/vtk/ttkAlgorithm/ttkAlgorithm.cpp

```diff
@@ -165,18 +150,20 @@ ttk::Triangulation *ttkAlgorithm::InitTriangulation(void *key,
     }
 
     // Cells
-    if(cells->GetNumberOfCells() > 0) {
-#ifdef TTK_CELL_ARRAY_NEW
+    int nCells = cells->GetNumberOfCells();
+    if(nCells > 0) {
       auto connectivity
         = (vtkIdType *)ttkUtils::GetVoidPointer(cells->GetConnectivityArray());
       auto offsets
         = (vtkIdType *)ttkUtils::GetVoidPointer(cells->GetOffsetsArray());
-      triangulation->setInputCells(
-        cells->GetNumberOfCells(), connectivity, offsets);
-#else
-      triangulation->setInputCells(
-        cells->GetNumberOfCells(), cells->GetData()->GetPointer(0));
-#endif
+      int status = triangulation->setInputCells(nCells, connectivity, offsets);
```

Comment:
Is this ParaView 5.7 compliant ? I am afraid the `setInputCells` method with two parameters may not work for older versions